src/lib/port-finder.ts

```diff
@@ -1,4 +1,5 @@
 import http from 'http';
+import killPort from 'cross-port-killer';
```

Comment:
Before deciding to use this library, I tried a few things to try to release a `CLOSED` port for use, including: 

- Used `http.createServer` to attempt to `.close` the port explicitly. This led to the port remaining in a `CLOSED` state, but not being released, which is the same as what's already happening when a site is deleted ([reference](https://github.com/Automattic/studio/blob/8fc3a5dd024a194f016a0a8e286be19de84c32af/vendor/wp-now/src/start-server.ts#L86-L92)).
- Experimented with explicitly setting `socket.setReuseAddress(true)`, so as to be able to re-use the port more quickly and also detecting/destroying open sockets.

The only success I had was with manually killing the PIDs associated with a given port using a `lsof` command, which is only available for Unix systems. The `cross-port-killer` library works for all platforms and using it felt more efficient than the code that would be required.

I'm open to further thoughts or ideas here.

Reply:
Thanks for explaining the reasoning behind using the library for that. It seems that the library is a wrapper and uses tools available in win32, linux, and darwin to kill the port, so it would go well with our support for Mac, Windows, and Linux in the future.